### DISCUSSION TITLE
The legend does not reflect my Stroke color

### DISCUSSION THREAD

**Discussion by Daimonion1980** [Live-Charts/LiveCharts2#1427] [Q&A] [answered]
Hello Community. 

Maybe someone can give me a hint, why my stroke color is not reflected in the legend.
Please take a look at the screenshot:

![image](https://github.com/beto-rodriguez/LiveCharts2/assets/12880413/54f4c868-04f7-4863-a2a4-d895c94c7293)

And here are the codesnippets, I use to display the series:

ViewModel:
```csharp
partial void OnInitialize()
{
    //temporäre initalisierung einer Achse, des Plots
    PlotData = new ObservableCollection<ISeries>
    {
        new LineSeries<ObservableValue>
        {
            Values = plotValue1,
            Fill = null,
            GeometrySize = 0,
            Stroke = new SolidColorPaint(SKColors.Red),
            LineSmoothness = 0.3,
            Name="Drehzahl rpm"
        }
    };
}
```

UserControl:
```xml
<Grid>
    <lvc:CartesianChart Grid.Row="0" Height="Auto" Series="{Binding PlotData}" 
                        ZoomMode="ZoomX" LegendPosition="Bottom" />
</Grid>
```

I found [issue 365](https://github.com/beto-rodriguez/LiveCharts2/issues/365) but this should already be fixed in 2.0.0-rc2 ?!

Does anybody see the problem?

**Comment by beto-rodriguez** [comment, marked as answer]
See https://github.com/beto-rodriguez/LiveCharts2/issues/1507 for the response.

**Reply by Daimonion1980** [reply]
Thanks for your hint!
